src/test/java/com/statsig/androidsdk/TestSharedPreferences.kt

```diff
@@ -0,0 +1,50 @@
+package com.statsig.androidsdk
+
+import android.content.SharedPreferences
+
+class TestSharedPreferences : SharedPreferences {
```

Comment:
Creating test object instead of mocking everything out. This can come in handy for setting state for tests later on.